### PR TITLE
Add volto-slate-label components

### PR DESCRIPTION
**Pull request by nileshgulia1**
## Edit mode:
Screenshot 2022-03-28 at 5 17 51 PM

Screenshot 2022-03-28 at 5 20 45 PM

## View:
Screenshot 2022-03-28 at 5 18 09 PM

### CODE REVIEW

File: src/editor/styles.less [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.ui.popup.high {
```
Comment (nileshgulia1) [inline src/editor/styles.less:1]: @avoinea How about making these styles configurable? For now these are taken from https://github.com/eea/volto-eea-design-system/blob/develop/theme/themes/eea/elements/label.overrides#L10

**Comment by avoinea** [comment]
no-type

@nileshgulia1 
I not type is selected, the popup is white background with white text on it. You should default one of the options.

**Comment by avoinea** [comment]
@nileshgulia1 You should also add another switch for `Always open` in the sense that the tooltip box should be closed by default and appear when you click on the selected text. If `Always open` is `true` then the tooltip should stay always opened. The text in tooltip box should be `richtext` (we should be able to add links and sub sup formating CO₂, etc.)

File: src/editor/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import mentainanceSVG from '@plone/volto/icons/maintenance.svg';
```
[the remark this answers is not in the document]
Comment (avoinea) [reply src/editor/index.js:1]: ...and `comment.svg` :smile:

[the remark this answers is not in the document]
Comment (avoinea) [reply src/editor/index.js:1]: See https://pastanaga.io/icons/

**Review by avoinea — CHANGES_REQUESTED** [review]

File: src/editor/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (nileshgulia1) [reply src/editor/index.js:1]: > Can you try `annotation.svg` icon.

mm.. I too want `annotation` 😭 https://github.com/plone/volto/issues/3207#issuecomment-1081790665

**Review by avoinea — APPROVED** [review]

**Pull request merged**